.circleci/config.yml

```diff
@@ -403,6 +401,8 @@ jobs:
     working_directory: ~/repo
     docker: *ref_0
     resource_class: large
+    environment:
+      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
```

Comment:
I don't think this env var should be set. It is only intended for testing the packaged CLI. The tests should be able to find the node CLI without it

Reply:
It does not pick up the right binary and the migration tests right now are broken if you actually point it 4.0.0

Reply:
ah yeah I see you're using it to migration test. that's fine then